output/print_methods.go

```diff
@@ -2,59 +2,74 @@ package output
 
 import (
 	"fmt"
+	"net/http"
 	"strings"
 
 	"github.com/fatih/color"
 	"github.com/visola/go-http-cli/request"
 )
 
-type bodyBuffer struct {
-	*strings.Reader
-}
-
-func (bb *bodyBuffer) Close() error {
-	return nil
-}
-
 // PrintRequest outputs the http.Request
-func PrintRequest(request request.Request) error {
+func PrintRequest(request request.Request) {
 	boldGreen := color.New(color.Bold, color.FgGreen).PrintfFunc()
 	boldGreen("\n%s %s\n", request.Method, request.URL)
 
-	sentHeaderKeyColor := color.New(color.Bold, color.FgBlack).PrintfFunc()
-	sentHeaderValueColor := color.New(color.FgBlack).PrintfFunc()
+	printHeaders(request.Headers)
+	printCookies(request.Cookies)
+	printBody(request.Body, ">>")
+}
 
-	for headerName, values := range request.Headers {
-		sentHeaderKeyColor("%s:", headerName)
-		if len(values) > 1 {
-			for _, val := range values {
-				sentHeaderValueColor("\n  %s", val)
-			}
-			fmt.Println("")
-		} else {
-			sentHeaderValueColor(" %s\n", values[0])
+// PrintResponse outputs a http.Response
+func PrintResponse(response request.Response) {
+	if response.StatusCode < 300 {
+		color.Green("%s %s\n", response.Status, response.Protocol)
+	} else if response.StatusCode < 400 {
+		color.Yellow("%s %s\n", response.Status, response.Protocol)
+	} else {
+		color.Red("%s %s\n", response.Status, response.Protocol)
+	}
+
+	printHeaders(response.Headers)
+	printBody(response.Body, "<<")
+}
+
+func printBody(body string, linePrefix string) {
+	if body != "" {
+		bodyColor := color.New(color.Bold).PrintfFunc()
+		split := strings.Split(body, "\n")
+		for _, line := range split {
```

Comment:
if you wanted to be even more terse, you could move the previous line into this `for` like
```golang
for _, s := range strings.Split("foo\nbar\n", "\n")
```

Reply:
Yeah, I don't need that `split` variable.